src/main/java/mate/academy/service/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.service;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://127.0.0.1:3306/test";
```

Comment:
The `DB_URL` is incomplete. It should contain the full JDBC URL to connect to the database. For example, "jdbc:mysql://localhost:3306/myDatabase", where `myDatabase` is the name of your database.